File: Craqudos.Dominio/Entidades/Equipe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Craqudos.Dominio.Entidades
{
    public class Equipe : Entidade
    {
        public int Id { get; set; }
        public DateTime DataSorteio { get; set; }
        public int UsuarioId { get; set; }
        public virtual Usuario Usuario { get; set; }
        public virtual ICollection<JogadorEquipe> JogadoresEquipe { get; set; }

        public override void Validar()
        {
            //if (!Jogadores.Any())
            //    AdicionarExcecao("Uma equipe deve possuir ao menos um jogador.");
        }
    }
}

File: Craqudos.Web/ClientApp/src/app/modelo/equipe.ts
import { JogadorEquipe } from "./jogadorEquipe";

export class Equipe {
  public id: number;
  public dataSorteio: Date;
  public configuracaoId: number;
  //public usuarioId: number;
  public jogadoresEquipe: JogadorEquipe[];

  constructor() {
    this.jogadoresEquipe = [];
  }
}
